packages/shell-evaluator/src/shell-evaluator.spec.ts

```diff
@@ -87,15 +87,15 @@ describe('ShellEvaluator', function () {
       expect(itSpy).to.have.been.calledWith();
     });
 
-    it('forwards the exit/quit command', async function () {
+    it('forwards the exitquit command (exit)', async function () {
```

Comment:
```suggestion
    it('forwards the exit/quit command (exit)', async function () {
```

Reply:
oops. was going to make then forwards the exit command and forwards the quit command but then didn't undo all the typing :)